### PR TITLE
[ML] Add 12-01-2022-preview swagger to ML Client

### PR DESCRIPTION
**Pull request by nthandeMS**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by TonyJ1 — APPROVED** [review]
:shipit:

File: sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/registries.json [hunk 1]
```diff
@@ -0,0 +1,874 @@
...
+          "type": "string"
+        },
+        "tier": {
+          "description": "This field is required to be implemented by the Resource Provider if the service has more than one tier, but is not required on a PUT.",
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/SkuTier"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Registry": {
```
Comment (MilesHolland) [inline sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/registries.json:510]: It appears that one of my comments was missed in the attempts to fix this API. The managedResourceGroupTags is missing, as I specified in the discussion of this relevant task: https://dev.azure.com/msdata/Vienna/_workitems/edit/2047140/

Can you please copy the change to registries.json that occurred in this PR, then regenerate the swagger please. If not, then I suspect that any accidental attempts to use registries with the new API will either explode, or worse, fail silently.

https://github.com/Azure/azure-sdk-for-python/pull/26977/files#diff-3a56962ce5e7fd7bd3fdbbfde0df665e90d34b8c5468f2aadcb21df98a68f9fb

**Review by MilesHolland — CHANGES_REQUESTED** [review]

File: sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/registries.json (hunk 1 above)
Comment (luigiw) [reply sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/registries.json:510]: @MilesHolland  Thx for the catch.

I hope we don't need to do this manual edit with every new API version. The work item you linked is closed, does that mean the backend service fix is done?

Comment (MilesHolland) [reply sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/registries.json:510]: The work item I mentioned was supposed to include a fix for this problem, but for some reason my comments in that task's PR regarding this missing piece were ignored, and the PR was merged and the task closed. I'm going to open up a new task to address this issue specifically for the next API release.

Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/registries.json:510]: Just added this in, check it out.

**Review by MilesHolland — APPROVED** [review]

**Pull request merged**